### ISSUE TITLE
Procedural catenaries

### ISSUE THREAD

**Issue by Insprill** [Insprill/dv-mapify#9]
The `MapInfo` component should have a field for a catenary pole prefab, two values for the upper/lower anchor point of the wires, and the spacing of the poles. Each track would then have a toggle for whether catenaries are generated on them, and if they are, the poles and wires will be generated either at build time in the editor or at runtime. The spacing of the poles would need to be aware of connected tracks to ensure consistent spacing.

**Comment by Insprill** [comment]
Closing since there's no point in us putting in a bunch of work for this if an official system is coming soon™.

**Comment by WhistleWiz** [comment]
Yeah and having the catenary would require an actual electric simulation, which would be outside the scope of the mod. For simple props it can already be done.
